tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusMolliePlugin\PHPUnit\Functional\Api;
+
+use BitBag\SyliusMolliePlugin\Client\MollieApiClient;
+use BitBag\SyliusMolliePlugin\Entity\OrderInterface;
+use BitBag\SyliusMolliePlugin\Repository\CreditMemoRepositoryInterface;
+use BitBag\SyliusMolliePlugin\Repository\OrderRepositoryInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use Payum\Core\Model\Identity;
+use Sylius\Component\Core\Model\Payment;
+use Sylius\Component\Core\Model\PaymentInterface;
+use Sylius\Component\Core\Repository\PaymentRepositoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\RefundPlugin\Entity\CreditMemoInterface;
+use Sylius\RefundPlugin\Entity\LineItemInterface;
+use Sylius\RefundPlugin\Entity\RefundInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\BitBag\SyliusMolliePlugin\PHPUnit\Functional\FunctionalTestCase;
+
+final class RefundOrderWebhookTest extends FunctionalTestCase
+{
+    /** @var MollieApiClient */
+    private $mollieApiClient;
```

Comment:
Could You change all fields to typed properties?

Reply:
Done :)